src/Rings/groebner/general.jl

```diff
@@ -240,3 +240,10 @@ function groebner_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_orde
     is_global(ordering) || error("Ordering must be global")
     return standard_basis(I, ordering=ordering, complete_reduction=complete_reduction, algorithm=algorithm)
 end
+
+function is_known(I::MPolyIdeal, ::typeof(groebner_basis); 
+    ordering::Union{MonomialOrdering, Nothing}=nothing)
```

Comment:
Could/should this also accept the `complete_reduction` kwarg?

Reply:
That depends on the user's need. In principle, I think, it could be added (if we agree on supporting forwarding of kwargs for `is_known`). But as this is just a demonstration right now, I restricted to this signature for the time being.